models/Settings.php

```diff
@@ -28,30 +28,30 @@ class Settings extends Model
 
     public function initSettingsData()
     {
-        $this->require_activation = config('rainlab.user::requireActivation', true);
-        $this->activate_mode = config('rainlab.user::activateMode', self::ACTIVATE_AUTO);
-        $this->use_throttle = config('rainlab.user::useThrottle', true);
-        $this->block_persistence = config('rainlab.user::blockPersistence', false);
-        $this->allow_registration = config('rainlab.user::allowRegistration', true);
-        $this->login_attribute = config('rainlab.user::loginAttribute', self::LOGIN_EMAIL);
-        $this->remember_login = config('rainlab.user::rememberLogin', self::REMEMBER_ALWAYS);
-        $this->use_register_throttle = config('rainlab.user::useRegisterThrottle', true);
+        $this->require_activation = config('winter.user::requireActivation', true);
+        $this->activate_mode = config('winter.user::activateMode', self::ACTIVATE_AUTO);
+        $this->use_throttle = config('winter.user::useThrottle', true);
+        $this->block_persistence = config('winter.user::blockPersistence', false);
+        $this->allow_registration = config('winter.user::allowRegistration', true);
+        $this->login_attribute = config('winter.user::loginAttribute', self::LOGIN_EMAIL);
+        $this->remember_login = config('winter.user::rememberLogin', self::REMEMBER_ALWAYS);
+        $this->use_register_throttle = config('winter.user::useRegisterThrottle', true);
```

Comment:
@jaxwilko @mjauvin @bennothommo should we look at adding aliasing support for config files?

Reply:
Yeah, would probably easy transition from October. Might also be a handy feature for devs going forwards too :)

Reply:
Might be more trouble than it's worth here. People wouldn't have changed these config items unless they published the configs. It'd be a simple rename of the config files/directories  in those cases and their configs would continue to work.

Reply:
What sort of trouble are you foreseeing @bennothommo? Also another thing that would be handy to alias is the october: commands, I keep on finding myself typing october:* instead of winter:* out of habit. Any thoughts on that?

Reply:
I agree, when I extend a plugin, I often use their config files as much as I can in order to ease maintenance. Aliasing the config files access would be a good idea if we want a smooth transition for plugin developpers.

Reply:
@LukeTowers it's only a micro-optimisation at best, but it would be better to simply check one file as opposed to two when getting config info for the plugin. I think the Config repository already reads all config files anyway so it's probably a moot point. So yeah, I take back my hesitance.